cx_Freeze/command/bdist_dmg.py

```diff
@@ -0,0 +1,353 @@
+            if self.icon:
+                add_param("icon", self.icon)
+            # We don't need to set this, as we only support icns
+            # add param ( "badge_icon", self.badge_icon)
+
+            # Window Settings
+            add_param("background", self.background)
+            add_param("show_status_bar", self.show_status_bar)
+            add_param("show_tab_view", self.show_tab_view)
+            add_param("show_pathbar", self.show_path_bar)
+            add_param("show_sidebar", self.show_sidebar)
+            add_param("sidebar_width", self.sidebar_width)
+            if self.window_rect:
+                add_param("window_rect", self.window_rect)
+            if self.default_view:
+                add_param("default_view", self.default_view)
+
+            add_param("show_icon_preview", self.show_icon_preview)
+            add_param(
+                "include_icon_view_settings", self.include_icon_view_settings
+            )
+            add_param(
+                "include_list_view_settings", self.include_list_view_settings
+            )
+
+            # Icon View Settings\
+            add_param("arrange_by", self.arrange_by)
+            add_param("grid_offset", self.grid_offset)
+            add_param("grid_spacing", self.grid_spacing)
+            add_param("scroll_position", self.scroll_position)
+            add_param("label_pos", self.label_pos)
+            if self.text_size:
+                add_param("text_size", self.text_size)
+            if self.icon_size:
+                add_param("icon_size", self.icon_size)
+            if self.icon_locations:
+                add_param("icon_locations", self.icon_locations)
+
+            # List View Settings
+            if self.list_icon_size:
+                add_param("list_icon_size", self.list_icon_size)
+            if self.list_text_size:
+                add_param("list_text_size", self.list_text_size)
+            if self.list_scroll_position:
+                add_param("list_scroll_position", self.list_scroll_position)
+            add_param("list_sort_by", self.list_sort_by)
+            add_param("list_use_relative_dates", self.list_use_relative_dates)
+            add_param(
+                "list_calculate_all_sizes", self.list_calculate_all_sizes
+            )
+            if self.list_columns:
+                add_param("list_columns", self.list_columns)
+            if self.list_column_widths:
+                add_param("list_column_widths", self.list_column_widths)
+            if self.list_column_sort_directions:
+                add_param(
+                    "list_column_sort_directions",
+                    self.list_column_sort_directions,
+                )
+
+            # License Settings
+            add_param("license", self.license)
+
+        print("\n\n\n\n")
+        dmgargs = [
+            "dmgbuild",
+            "-s",
+            "settings.py",
+            self.volume_label,
+            self.dmg_name,
+        ]
```

Comment:
How about using import?
on top:
`from dmgbuild.core import build_dmg`
here:
`build_dmg(self.dmg_name, self.volume_label, "settings.py")`

Reply:
Resolved in a9e0ed9f2f34b4b3aab4a610de31238a94ae6741